content/en/docs/collector/resiliency.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Resiliency
+description: How to configure a resilient OTel Collector pipeline
+---
+
+The OpenTelemetry Collector is designed with components and configurations to
+minimize data loss during telemetry processing and exporting. However,
+understanding the potential scenarios where data loss can occur and how to
+mitigate them is crucial for a resilient observability pipeline.
+
+## Understanding Collector resilience
+
+A resilient Collector maintains telemetry data flow and processing capabilities
+even when facing adverse conditions, ensuring the overall observability pipeline
+remains functional.
+
+The Collector's resilience primarily revolves around how it handles data when
+the configured endpoint (the destination for traces, metrics, or logs) becomes
+unavailable or when the Collector instance itself experiences issues like
+crashes.
+
+## Sending queue (in-memory buffering)
+
+The most basic form of resilience built into the Collector's exporters is the
+sending queue.
+
+- How it works: When an exporter is configured, it usually includes a sending
```

Comment:
nit: You could use a [definition list](https://michelf.ca/projects/php-markdown/extra/#def-list) for this, this seems to be supported by Hugo per [this page](https://gohugo.io/configuration/markup/#extensions), although I don't if you can do nested definition lists